doc/build-unix.md

```diff
@@ -134,9 +134,9 @@ Optional:
 
     sudo dnf install miniupnpc-devel
 
-To build with Qt 5 you need the following:
+To build elements-qt you need the following:
 
-    sudo dnf install qt5-qttools-devel qt5-qtbase-devel protobuf-devel
+    sudo dnf install qt5-qttools-devel qt5-qtbase-devel protobuf-devel ImageMagick librsvg2-tools
```

Comment:
Is this something that is also added upstream? Or is this needed for us because of the icon changes that Luke introduced?

Reply:
This is needed because of the icon changes. These are removed/reverted by #935. Let's please not dig this hole any deeper here.